examples/favorites_advanced/favorites_advanced_base/pubspec.yaml

```diff
@@ -11,22 +11,24 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  skeleton_text:
+  skeleton_text: ^3.0.0
+#  skeleton_text:
   visibility_detector: ^0.2.0-nullsafety.1
   connectivity: ^3.0.2
   image_picker: ^0.7.0
   collection: ^1.15.0-nullsafety.4
   font_awesome_flutter: ^9.0.0
   intl: ^0.17.0
-  smooth_star_rating: ^1.1.1
+#  smooth_star_rating: ^1.1.1
```

Comment:
@StanislavPrime, by removing this package you are breaking the booking app. What is the reason of removing it ?

Reply:
The dependency was uncommented